style.css

```diff
@@ -391,7 +402,145 @@ hr {
   margin-left: 32px;
 }
 
-@media only screen and (max-width: 768px) {
+/* validate */
+
+.error-message {
+  color: red;
+  font-size: 17px;
+  align-self: center;
```

Comment:
## [OPTIONAL]

- [ ] Kindly I would recommend you to work on your desktop version. Make sure your design looks like the template's one, refer here down 👇  👇 . 

https://user-images.githubusercontent.com/65068771/146641742-ebbab29a-043d-4907-abd6-7a340b573b7b.mp4